default_warehouse_from_sale_team/__openerp__.py

```diff
@@ -15,6 +15,7 @@
         'sales_team',
         'base_action_rule',
         'purchase_requisition',
+        'warning',
```

Comment:
The dependence to website module was remove in [9e00310](https://github.com/Vauxoo/addons-vauxoo/pull/1060/commits/9e003100c158aac6f2d69b8f6ac6434e17b39443)